### PR TITLE
Added a note to distributing.rst about pure Python wheels

### PR DESCRIPTION
**Pull request by pfmoore**

### CODE REVIEW

File: source/distributing.rst [hunk 1]
```diff
@@ -478,13 +478,44 @@ and requires a build step when installed by pip.  Even if the distribution is
 pure python (i.e. contains no extensions), it still involves a build step to
 build out the installation metadata from ``setup.py``.
 
+.. _`Wheels`:
+
+Wheels
+------
+
+You should also create a wheel for your project. A wheel is an installable
+binary package of your project which can be installed without needing to go
```
Comment (qwcode) [inline source/distributing.rst]: better to say "built".  the glossary distinguishes "built" vs "binary"

File: source/distributing.rst [hunk 2]
```diff
@@ -478,13 +478,44 @@ and requires a build step when installed by pip.  Even if the distribution is
...
+
+ python setup.py bdist_wheel
+
+Assuming that your project does not contain any C extensions, this will build
+a "Pure Python Wheel". It will be usable on any Python installation with the
+same major version (Python 2 or Python 3) as the version you used to build
+the wheel.
+
+If your project will run on both Python 2 and Python 3 with the same source,
+you should produce a "Universal Wheel" (see below). If your project includes
```
Comment (qwcode) [inline source/distributing.rst]: this is vague.  it almost sounds like a universal wheel is in addition to the version-specific wheels.

File: source/distributing.rst [hunk 3]
```diff
@@ -478,13 +478,44 @@ and requires a build step when installed by pip.  Even if the distribution is
...
+C extensions, you should produce "Platform Wheels" for the platforms you
+support.
```
Comment (qwcode) [inline source/distributing.rst]: you don't reference the section below

**Comment by qwcode** [comment]
@pfmoore if you don't mind, let me try to restructure this.   I guess the goals here were:

1. to restate the benefit of wheels
2. to mention pure non-universal wheels, which weren't covered before.

**Comment by qwcode** [comment]
ok, see PR #137

**Comment by pfmoore** [comment]
@qwcode thanks, that's much better than my attempt.

**Pull request closed without merging**